### ISSUE TITLE
Ghost flares hovering above vessel after parachute landing

### ISSUE THREAD

**Issue by Sethur** [MOARdV/DistantObject#21]
Hi,

with KSP v1.0.4 using OpenGL and DOE v1.6.1, I get the strange ghost flares again, this time hovering the vessel after a standard parachute landing, as seen in the screenshots below. This happened two times in a row now, I think it's reproducable. Strange enough, I had flares deactivated  in the DOE settings the whole time.

![screenshot0](https://cloud.githubusercontent.com/assets/2142589/9618680/d6d6a134-5108-11e5-82e1-8e8bc78b1773.jpg)

![screenshot1](https://cloud.githubusercontent.com/assets/2142589/9618623/5b5363a8-5108-11e5-8734-813a72c80e5c.jpg)

Thanks for your efforts!

**Comment by Sethur** [comment]
Update: It seems that the ghost flares appear not only above the vessel after a parachute landing but also at random locations under or next to the vessel. This has happened at different times in the mission. Can anybody confirm this? I will try to add more screenshots of this when it happens again. I'm also suspecting that this is an OpenGL issue and that it does not happen when using DirectX but I haven't confirmed this yet.

**Comment by MOARdV** [comment]
I have never seen this bug.  It may be an interaction with one of the mods you're using, although, from what I can see in the screen shot, you've got KER (which I use) and FAR (which shouldn't be a problem - I use it occasionally, although I also use RealChutes, not the "light" version included in FAR) on your app bar.  Without additional specifics (what other mods are installed, logs from a session that includes the bug), I don't have a path forward to investigate the bug.

**Comment by Sethur** [comment]
Hi MOARdV, 

thanks for looking into this. I can't seem to append a text file so I have to do it like this: My list of current mods is:

******* START ********

"name": "AstronomersPack",
"version": "Interstellar.V2"
"name": "AstronomersPack-AtmosphericScattering",
"version": "Interstellar.V2"
"name": "AstronomersPack-Auroras-4K",
"version": "Interstellar.V2"
"name": "AstronomersPack-Clouds-Medium",
"version": "Interstellar.V2"
"name": "AstronomersPack-Eve-Jool-Clouds-4K",
"version": "Interstellar.V2"
"name": "AstronomersPack-Sandstorms",
"version": "Interstellar.V2"
"name": "AstronomersPack-SurfaceGlow",
"version": "Interstellar.V2"
"name": "AstronomersPack-DistantObjectEnhancement",
"version": "Interstellar.V2"
"name": "ModuleManager",
"version": "2.6.8"
"name": "AstronomersPack-PlanetShine",
"version": "Interstellar.V2"
"name": "PlanetShine",
"version": "0.2.3.1"
"name": "EnvironmentalVisualEnhancements",
"version": "7-4"
"name": "RasterPropMonitor",
"version": "v0.23.1"
"name": "RCSBuildAid",
"version": "0.7.2"
"name": "VesselView",
"version": "v0.71"
"name": "RasterPropMonitor-Core",
"version": "v0.23.1"
"name": "FerramAerospaceResearch",
"version": "3:0.15.5.1"
"name": "Toolbar",
"version": "1.7.9"
"name": "EditorExtensions",
"version": "2.12"
"name": "KerbalEngineerRedux",
"version": "1.0.18.0"
"name": "QuickScroll",
"version": "v1.31"
"name": "QuickSearch",
"version": "v1.13"
"name": "DistantObject",
"version": "v1.6.2"
"name": "ModularFlightIntegrator",
"version": "1.1.1"
"name": "DeadlyReentry",
"version": "v7.2.2"
"name": "QuickContracts",
"version": "v1.02"
"name": "QuickGoTo",
"version": "v1.20"
"name": "WaterSounds",
"version": "1.1"
"name": "TextureReplacer",
"version": "2.4.10"

****** END ******

As already stated, I'm using KSP 1.0.4 32bit on Windows with "-popupwindow -force-opengl" (which is due to much lower ram usage a very popular configuration).

Some more testing with this bug turned out that the ghost flares are sometimes also "released" from the vessel during orbiting. They then disappear quickly behind the ship (as if standing still in space, while the ship is moving away). I wasn't able to take a screenshot yet, because they are very quickly out of range.

What I also realized was that I had "flares" turned of in DOE, that means this particular texture should not appear in the first place. I now turned it on, and -- interestingly -- I did not come across the error since then. It might be that one has to have the flares setting "disabled" and use OpenGL to provoke this error.
